builtin/rebase.c

```diff
@@ -376,6 +377,18 @@ static int run_rebase_interactive(struct rebase_options *opts,
 	flags |= opts->rebase_cousins > 0 ? TODO_LIST_REBASE_COUSINS : 0;
 	flags |= command == ACTION_SHORTEN_OIDS ? TODO_LIST_SHORTEN_IDS : 0;
 
+	if (opts->ignore_whitespace) {
+		struct strbuf buf = STRBUF_INIT;
+
+		if (opts->strategy_opts)
+			strbuf_addstr(&buf, opts->strategy_opts);
+
+		strbuf_addstr(&buf, " --ignore-space-change");
+		free(opts->strategy_opts);
+		opts->strategy_opts = strbuf_detach(&buf, NULL);
+		strbuf_release(&buf);
```

Comment:
This `strbuf_release` is actually unnecessary.  From the comments for `strbuf_detach` in `strbuf.h`:

```
 * The strbuf that previously held the string is reset to `STRBUF_INIT` so
 * it can be reused after calling this function.
```

and from `strbuf_release`:

```
 * strbuf points to an empty string that does not need to be free()ed, as
 * if it had been set to `STRBUF_INIT` and never modified.
```

So a strbuf set to `STRBUF_INIT` doesn't need to be released, so neither does one after `strbuf_detach` (not that it will hurt much though).